src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub struct SleepParams {
     pub min_sleep_period: u64,
```

Comment:
Maybe this fields can be Duration ???

Reply:
it's a great idea to use Duration. what do you think about using duration for all fields of the struct

Reply:
[SleepParams Duration :D](https://github.com/ayrat555/fang/pull/67/commits/c82180cc1440a7833ee93c45b5f4a7aac6115b19)